joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/passes/AstCreator.scala

```diff
@@ -357,6 +377,73 @@ class AstCreator(filename: String, javaParserAst: CompilationUnit, global: Globa
     }
   }
 
+  private def substituteTypeVariable(resolvedType: ResolvedType, typeParamValues: ResolvedTypeParametersMap): String = {
+    substituteTypeVariableInternal(resolvedType, typeParamValues)
+  }
```

Comment:
Is this wrapper method necessary? Given that both `substituteTypeVariable` and `substituteTypeVariableInternal` are private, they seem to be equivalent.